libraries/AP_HAL_SITL/SITL_cmdline.cpp

```diff
@@ -229,8 +229,8 @@ void SITL_State::_parse_command_line(int argc, char * const argv[])
         switch (opt) {
         case 'w': {
             printf("Erasing EEPROM...\n");
+            unlink("eeprom.bin");
             AP_Param::erase_all();
```

Comment:
This is not needed, if the file is unlinked there is no parameters to erase.

Reply:
Yes maybe it isn't need to write the first header here